Documentation/markdown/SetupHelp.md

```diff
@@ -1,58 +1,45 @@
 # OpenInfraPlatform Installation 
 
 ***
-## Steps before the Installation 
+## Prerequisites 
 
 Before you can proceed to the Installation of **OpenInfraPlatform**, there are several steps, which have to be done before Installation: 
 
 1. Installation of **Visual Studio 2017** - 
 2. Installation of **CMake** -
 3. Installation of **Git** - 
 4. Creation of **GitHub account**
```

Comment:
This point should be as a prerequisite in the GitWorkflow.md.

Reply:
Done